packages/compass-indexes/src/modules/search-indexes.ts

```diff
@@ -633,7 +633,7 @@ export const dropSearchIndex = (
         title: `Your index ${name} is being deleted.`,
         dismissible: true,
         timeout: 5000,
-        variant: 'success',
+        variant: 'progress',
```

Comment:
Slightly unrelated, would a better flow for the toasts to be showing the `Your index x is being deleted...` until either it completes or an error occurs? It would make the flow similar to how the import/export toasts work: https://github.com/mongodb-js/compass/blob/d87a58e0817aa475e5e4c7c607e65a35c01b6d93/packages/compass-import-export/src/modules/export.ts#L423  https://github.com/mongodb-js/compass/blob/d87a58e0817aa475e5e4c7c607e65a35c01b6d93/packages/compass-import-export/src/components/export-toast.tsx#L52
They share the same id so they are overridden when the error or completion state happens.


Maybe this has been asked before, but does the `await dataService.dropSearchIndex()` method not indicate completion when the function succeeds? Should we be showing this toasat before that function call?

Reply:
It doesn't because for create, update and delete of search indexes the command (and therefore method) returns immediately and then the action gets "queued" and it can take any amount of time to complete.

Reply:
(Which is why we poll in the list view to keep updating the statuses.)